binding/dotnet/Leopard/Leopard.cs

```diff
@@ -48,20 +48,27 @@ public class Leopard : IDisposable
 
         static Leopard()
         {
-#if NETCOREAPP3_1_OR_GREATER
+
+#if NETCOREAPP3_0_OR_GREATER
```

Comment:
Do you know why the min version of .Net core was 3.1 instead of 3.0 before?

Reply:
It was a mistake - DllImportResolver has always been supported in Core 3.0+

Reply:
We should also reflect this in our docs then